www/docs/api/key.php

```diff
@@ -62,31 +72,44 @@
 
 # ---
 
-function create_key($commercial, $reason) {
+function create_key($commercial, $reason, $estimated_usage) {
     global $THEUSER;
     $key = auth_ab64_encode(urandom_bytes(16));
     $db = new ParlDB;
     if ($commercial=='') $commercial = 0;
-    $db->query('INSERT INTO api_key (user_id, api_key, commercial, created, reason) VALUES
-        (:user_id, :key, :commercial, NOW(), :reason)', array(
+    $db->query('INSERT INTO api_key (user_id, api_key, commercial, created, reason, estimated_usage) VALUES
+        (:user_id, :key, :commercial, NOW(), :reason, :estimated_usage)', array(
         ':user_id' => $THEUSER->user_id(),
         ':key' => $key,
         ':commercial' => $commercial,
-        ':reason' => $reason
+        ':reason' => $reason,
+        ':estimated_usage' => $estimated_usage
         ));
 }
 
 function api_key_form() {
 ?>
 <br>
-<h3>Apply for a new key</h3>
+<h3>Get a new key</h3>
 <form action="/api/key" method="post">
-<p>Is your application for:
-<input id="non_comm" type="radio" name="commercial" value="0"> <label for="non_comm">Non-commercial use</label>
-<input id="comm" type="radio" name="commercial" value="1"> <label for="comm">Commercial use</label>
+<p>About you:<br/>
+<input id="non_comm" type="radio" name="commercial" value="0"> <label for="non_comm">an individual pursuing a non-profit project on an unpaid basis</label><br/>
+<input id="non_comm_2" type="radio" name="commercial" value="0"> <label for="non_comm_2">a registered charity</label><br/>
+<input id="comm" type="radio" name="commercial" value="1"> <label for="comm">neither of the above</label>
+</p>
 <p><label for="reason">Please describe what you're going to use the key for:</label>
 <br>
 <textarea id="reasons" name="reason" rows=7 cols=40></textarea>
+</p>
+<p>What's your estimated annual API call volume?<br/>
+    <input id="estimated_usage_50k" type="radio" name="estimated_usage" value="50000"> <label for="estimated_usage_50k">50,000</label><br/>
```

Comment:
Should this perhaps say "up to 50,000", or the question say 'maximum', otherwise if you know yours will be low usage you might not be sure which to pick.